pkg/action/action_test.go

```diff
@@ -289,3 +290,12 @@ func TestGetVersionSet(t *testing.T) {
 		t.Error("Non-existent version is reported found.")
 	}
 }
+
+func loadChart(t *testing.T, path string) *chart.Chart {
```

Comment:
this is a utils func, why has testing.T arg? you can return (*chart.Chart, error), WDYT?